docs/source/usage.rst

```diff
@@ -301,3 +301,48 @@ Settings related to the Kinto server
         }
       ]
     }
+
+
+How do I create the ``validate_changes_collection`` function on Amazon Lambda?
+------------------------------------------------------------------------------
+
+1. `Log into the Amazon Web Service console. <https://console.aws.amazon.com/lambda/home>`_
+2. Click the **Create a Lambda Function** button
+3. Skip the blueprint
+4. Configure the function: **name**, **description**
+5. Select **Python 3.6** runtime
+6. Select **Upload a file from Amazon S3**
+7. Enter your S3 ``lambda.zip`` URL
+8. Use **lambda_function.validate_changes_signature** handler
+9. Set a **Lambda S3 Exec Role**
+10. Configure a 5 minutes timeout
+11. You can proceed without a VPC
+
+
+How do I run the ``validate_signature`` function on Amazon Lambda?
+------------------------------------------------------------------
+
+Once your function has been configured, you can run it.
+
+To do that, select the function and in the **Actions** menu select
+**Configure Test event**.
+
+You will then be able to enter a **Input test event** as a JSON.
+
+This function reads the content and signature of a collection and
+validate the signature is valid.
+
+The ``validate_changes_collection`` let you define the following inputs:
+
```

Comment:
This is weird to end up here with a colon,  plus the next paragraph title *Settings related to the Kinto server* does not really make sense here..